client/scripts/DrawTogether.js

```diff
@@ -597,8 +599,16 @@ DrawTogether.prototype.changeRoom = function changeRoom (room, number, x, y, spe
 			this.paint.addPublicDrawings(this.decodeDrawings(drawings));
 			this.chat.addMessage("Invite people", "http://www.anondraw.com/#" + room + number);
 			
-			if (this.account.uKey) {
```

Comment:
i noticed that it this.account.uKey would exist and you would still not be logged in( like on playerserver restarts ) and that the reputation was always a clear indicator